types/define-properties/.editorconfig

```diff
@@ -0,0 +1,3 @@
+# This package uses tabs
+[*]
+indent_style = tab
```

Comment:
Is this necessary?

Reply:
Yes.

Otherwise, Prettier will&nbsp;convert tabs&nbsp;to&nbsp;spaces, which&nbsp;I&nbsp;don’t&nbsp;want.